scripts/createEthRollup.ts

```diff
@@ -3,16 +3,12 @@ import '@nomiclabs/hardhat-ethers'
 import { createRollup } from './rollupCreation'
 
 async function main() {
+  const feeToken = ethers.constants.AddressZero
```

Comment:
I got a conflict here. Since this is for creating an Eth rollup I think setting to zero always makes sense.